tests/sentry/event_manager/test_event_manager.py

```diff
@@ -1395,6 +1398,7 @@ def test_csp_event_type(self):
             "directive": "script-src",
             "uri": "example.com",
             "message": "Blocked 'script' from 'example.com'",
+            "sdk": "other",
```

Comment:
Aha! This brings up a really good point that I hadn't thought of: Not every event type has or should have an `sdk` value. CSP events, for example, are sent natively by the browser, not by the SDK. I think we should probably only add the tag when the type is right.

The possible types come from relay: https://github.com/getsentry/relay/blob/1bc9fc9830dee70ca2033f80e5c7082ed4c007fb/relay-base-schema/src/events.rs#L87-L101

and correspond to the the event types here: https://github.com/getsentry/sentry/blob/master/src/sentry/eventtypes/__init__.py#L12-L22

It's those latter ones you'll be checking against. The event manager modules has a `get_event_type` helper you can use.

There are a few types I'm not sure about (when are events generic? feedback events come from the SDK... I think?), so let's start by excluding the ones we know for sure aren't SDK-generated, which is to say,  all of the ones between `csp` and `expectstaple` in that list. If we see any of the other types coming in with nothing but `other`, we'll know we need to add to the list of excluded types.

Reply:
Done, I've used `SECURITY_REPORT_INTERFACES` constant for this (curiously this constant doesn't seem to be used anywhere at all).

Reply:
> `SECURITY_REPORT_INTERFACES`

TIL. We should add `nel` in there, too, which was added recently to our report types.